features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Default/APICreateDefault.jsx

```diff
@@ -49,6 +53,23 @@ function APICreateDefault(props) {
     const [pageError, setPageError] = useState(null);
     const [isCreating, setIsCreating] = useState();
     const [isPublishing, setIsPublishing] = useState(false);
+    const [policies, setPolicies] = useState([]);
+
+    useEffect(() => {
+        API.policies('subscription').then((response) => {
+            const allPolicies = response.body.list;
+            if (allPolicies.length === 0) {
+                Alert.info(intl.formatMessage({
+                    id: 'Apis.Create.Default.APICreateDefault.error.policies.not.available',
+                    defaultMessage: 'Throttling policies not available. Contact your administrator',
+                }));
+            } else if (allPolicies.filter((p) => p.name === 'Unlimited').length > 0) {
```

Comment:
Same here, We could use `.find()`, it will break when with first match and filter will iterate through the entire list right ?